app/models/marc_fields/awards.rb

```diff
@@ -0,0 +1,14 @@
+##
+# A class to handle MARC 586 field logic
+# https://www.loc.gov/marc/bibliographic/bd586.html
+class Awards < MarcField
+  private
+
+  def display_value(field, subfields)
+    safe_join [super, Constants::SOURCES[field['1']]].compact, subfield_delimeter
```

Comment:
Looks like this method has a typo in it and it appears in four places. Not a biggie, and this PR didn't add the method, but thought I'd point out `s/delimeter/delimiter/` :man_shrugging: